### DISCUSSION TITLE
virtio-fs/virtiofs vagrant message too verbose?

### DISCUSSION THREAD

**Discussion by rgl** [vagrant-libvirt/vagrant-libvirt#1488] [Q&A] [answered]
I'm trying virtiofs for the first time in ubuntu 22.04, and was pretty glad it worked! :-)

I'm using this in my `Vagrantfile`:

```ruby
lv.memorybacking :access, :mode => 'shared'
config.vm.synced_folder '.', '/vagrant', type: 'virtiofs'
```

So far, I only found this quirk, this vagrant message is way too verbose, and uses too much screen/output estate:

==> default:  /vagrant, opts: {:type=>:virtiofs, :guestpath=>"/vagrant", :hostpath=>"/home/vagrant/Projects/ubuntu-vagrant/example", :disabled=>false, :__vagrantfile=>true, :plugin=>#<VagrantPlugins::SyncedFolderVirtioFS::SyncedFolder:0x00007f5b9105bb78 @logger=#<Log4r::Logger:0x00007f5b9105bad8 @fullname="vagrant_libvirt::synced_folders::virtiofs", @outputters=[], @additive=true, @name="virtiofs", @path="vagrant_libvirt::synced_folders", @parent=#<Log4r::RootLogger:0x0000000002c84450 @level=0, @outputters=[]>, @level=0, @trace=false>, @cap_logger=#<Log4r::Logger:0x00007f5b9103bda0 @fullname="vagrant::capability_host::vagrantplugins::syncedfoldervirtiofs::syncedfolder", @outputters=[], @additive=true, @name="syncedfolder", @path="vagrant::capability_host::vagrantplugins::syncedfoldervirtiofs", @parent=#<Log4r::RootLogger:0x0000000002c84450 @level=0, @outputters=[]>, @level=0, @trace=false>, @cap_host_chain=[[:virtiofs, #<VagrantPlugins::SyncedFolderVirtioFS::SyncedFolder:0x00007f5b9173b240 @logger=#<Log4r::Logger:0x00007f5b9173b1a0 @fullname="vagrant_libvirt::synced_folders::virtiofs", @outputters=[], @additive=true, @name="virtiofs", @path="vagrant_libvirt::synced_folders", @parent=#<Log4r::RootLogger:0x0000000002c84450 @level=0, @outputters=[]>, @level=0, @trace=false>>]], @cap_args=[#<Vagrant::Machine: default (VagrantPlugins::ProviderLibvirt::Provider)>], @cap_caps={:smb=>#<Vagrant::Registry:0x00007f5b9103c228 @items={:default_fstab_modification=>#<Proc:0x00000000030f66f0 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/synced_folders/smb/plugin.rb:26>, :mount_options=>#<Proc:0x00000000030f65d8 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/synced_folders/smb/plugin.rb:31>, :mount_name=>#<Proc:0x00000000030f64e8 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/synced_folders/smb/plugin.rb:36>, :mount_type=>#<Proc:0x00000000030f63a8 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/synced_folders/smb/plugin.rb:41>}, @results_cache={}>, :virtualbox=>#<Vagrant::Registry:0x00007f5b9103c0e8 @items={:mount_options=>#<Proc:0x0000000002c6a7f8 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/providers/virtualbox/plugin.rb:72>, :mount_type=>#<Proc:0x0000000002c6a780 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/providers/virtualbox/plugin.rb:77>, :mount_name=>#<Proc:0x0000000002c6a690 /opt/vagrant/embedded/gems/2.2.19/gems/vagrant-2.2.19/plugins/providers/virtualbox/plugin.rb:82>}, @results_cache={}>}>, :target=>"/vagrant", :mount=>true, :readonly=>nil, :mount_tag=>"c76718fe5e3581624e62b6520910ffc"}

Is that normal?

Can it be omitted from the default output? Should I open an issue/PR?

**Comment by electrofelix** [comment]
That looks like there may be a stray 'puts' for debug left in there by accident, certainly there is no need for that level of detail to be emitted

**Comment by electrofelix** [comment, marked as answer]
Found the problem line

https://github.com/vagrant-libvirt/vagrant-libvirt/blob/c5ed4bd89c7a508825fda91ef3c86bd435b9be50/lib/vagrant-libvirt/cap/synced_folder_virtiofs.rb#L47

**Reply by electrofelix** [reply]
Ugh, nope that wasn't it, I suspect it's near there though

**Reply by electrofelix** [reply]
Actually it turns out that was exactly where it was, just the formatting threw me off.

@rgl this should be fixed with the next release.
